includes/Lifecycle.php

```diff
@@ -39,6 +39,7 @@ public function __construct( $plugin ) {
 			'1.10.0',
 			'1.10.1',
 			'1.11.0',
+			'2.0.0',
```

Comment:
I'd prefer to release this before `2.0.0`.

Reply:
I reverted the commits that change the version number with f6a1b29. We will need to revert others from the changes I merged yesterday, but we can do that in another PR after 1.11.3 is out.